base/task.jl

```diff
@@ -4,6 +4,49 @@
 
 show(io::IO, t::Task) = print(io, "Task ($(t.state)) @0x$(hex(convert(UInt, pointer_from_objref(t)), WORD_SIZE>>2))")
 
+type CollectedException <: Exception
+    pid::Int
+    ex
+    bt
```

Comment:
Sure, but why does specifying a pid always go along with specifying a backtrace? It's a bit random.